torchbenchmark/models/clip/requirements.txt

```diff
@@ -0,0 +1 @@
+git+https://github.com/facebookresearch/multimodal.git@bf42547c5fcfd063eba4f036def0c57309c2ee57
```

Comment:
why not just use the nightly?

Reply:
It's what was in your notebook, I just copied that - can use nightly if you prefer and if no breaking changes are expected

Reply:
i fixed